app/models/API1811/Error.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.API1811
+
+import play.api.libs.json.{Format, Json}
+
+case class Error(code: String, reason: String)
```

Comment:
We might need to reconsider what we're doing for this. If we want to read the error message from JSON, the reads will need to change as the errors appear in a `failures` array. If we just want to simply shove the whole response body in a class, perhaps "code" and "reason" are not appropriate params as you won't be able to parse those individual fields from the body without some messy String logic.

The simplest approach would be one field for the response body. Can we think of a reason why we would want more than that, or is that sufficient? Does that match with what Will was expecting? In the work he's done for the parser, does he expect to be reading the error JSON or simply constructing this class with the response body?